### ISSUE TITLE
Get function documentation

### ISSUE THREAD

**Issue by kflak** [martineausimon/nvim-lilypond-suite#31]
This might very well be a giant undertaking, but I am really missing a way to hit `K` on some keyword or function and get the documentation for this... Do you see any plausible way to accomplish this?

**Comment by martineausimon** [comment]
Good idea. I don't have much time at the moment to do this, but I'll think about it, and do it as soon as possible.
